### PR TITLE
Update vt_tutorial.py

### PR DESCRIPTION
**Pull request by brianjo**
Testing reformatting the vt_tutorial for better colab output.

### CODE REVIEW

**Comment by brianjo** [comment]
Preview of the notebook file: https://colab.research.google.com/drive/1WOhLan6w-_O6SQnGeMX4xvhowJ58VBOP?usp=sharing

Screen Shot 2021-06-28 at 12 20 40 PM

File: beginner_source/vt_tutorial.py [hunk 1]
```diff
@@ -50,17 +50,16 @@
...
-#    pip install pandas
-#    pip install requests 
-#
-# then run the script below:
-#
 
+# pip install torch torchvision timm pandas requests 
+
+#######################################################
+# To run in Google Colab, uncomment the following line:
```
Comment (jeffxtang) [inline beginner_source/vt_tutorial.py:57]: This is good if we can't find a better solution that will generate slightly different format about `#! pip install...` when converting the source Python script to rst and ipynb.

File: beginner_source/vt_tutorial.py [hunk 2]
```diff
@@ -50,17 +50,16 @@
...
-#
-#    pip install torch torchvision
-#    pip install timm
-#    pip install pandas
-#    pip install requests 
-#
-# then run the script below:
-#
 
+# pip install torch torchvision timm pandas requests
```
[the remark this answers is not in the document]
Comment (brianjo) [reply beginner_source/vt_tutorial.py:54]: One is an instruction for installing those dependencies locally, the other is just a note for enabling the sample in Colab. We're a little limited in how we can format content in colab and make it readable here as well.

File: beginner_source/vt_tutorial.py [hunk 3]
```diff
@@ -50,17 +50,16 @@
...
-# then run the script below:
-#
 
+# pip install torch torchvision timm pandas requests 
+
+#######################################################
+# To run in Google Colab, uncomment the following line:
+
+# !pip install timm pandas requests
+
```
[the remark this answers is not in the document]
Comment (brianjo) [reply beginner_source/vt_tutorial.py:60]: No, colab already comes with the PyTorch dependencies installed.

**Comment by jeffxtang** [comment]
Looks good to me Brian. Thanks!

**Comment by brianjo** [comment]
> Looks good to me Brian. Thanks!

Thanks for your input Jeff!

**Pull request merged**